libs/ledger-live-common/src/families/avalanchepchain/serialization.ts

```diff
@@ -0,0 +1,47 @@
+import BigNumber from "bignumber.js";
+import type {
+  AvalanchePChainResourcesRaw,
+  AvalanchePChainResources,
+} from "./types";
+
+export function toAvalanchePChainResourcesRaw(
+  r: AvalanchePChainResources
+): AvalanchePChainResourcesRaw {
+  const { publicKey, chainCode, stakedBalance, delegations } = r;
+
+  return {
+    publicKey,
+    chainCode,
+    stakedBalance: stakedBalance.toString(),
+    delegations: delegations?.map((delegation) => ({
+      txID: delegation.txID,
+      startTime: delegation.startTime,
```

Comment:
If i remind they are no date but just an unix timestamp in string right ?

Reply:
Yep, that's correct:

https://github.com/LedgerHQ/ledger-live/blob/cdb1c6d8aeb51d2393a9a42d454047c8a448ee60/libs/ledger-live-common/src/families/avalanchepchain/types.ts#L39